docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.row.scss

```diff
@@ -76,6 +76,10 @@
   }
 }
 
+.counted-items-1.field-hs-row-components {
```

Comment:
I would like to write a comment here too about how if there is just one item in the row, there is no need for flex